README.md

```diff
@@ -35,6 +40,15 @@ npm install
 npm run dev
 ```
 4. Go to http://localhost:8080
+
+## Create DB 
```

Comment:
Let's rename this to be something more descriptive: `How to setup the database?` and link it under the TOC also